### PR TITLE
Normalize line endings in SSH key for the underlying OS

### PR DESCRIPTION
**Pull request by mansandersson**
This fixes issue #143 on my side

This is my first ever nodejs commit so please help me out if I've done something completely unacceptable.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "https://github.com/easingthemes/ssh-deploy#readme",
   "dependencies": {
+    "os": "^0.1.2",
```
Comment (easingthemes) [inline package.json:33]: `os` is native nodeJS module, please remove it as a dependency

File: src/sshKey.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const { join } = require('path');
 const { execSync } = require('child_process');
 const { writeToFile } = require('./helpers');
+const { EOL } = require('os');
```
Comment (easingthemes) [inline src/sshKey.js:4]: Move line up in a group with other nodeJS modules

File: package-lock.json
Comment (easingthemes) [inline package-lock.json:1]: After removing dependencies there should be no changes to package.lock.

**Review by easingthemes — CHANGES_REQUESTED** [review]

**Comment by codespearhead** [comment]
@mansandersson are you still willing to act on the changes and have this PR merged?

**Comment by mansandersson** [comment]
> @mansandersson are you still willing to act on the changes and have this PR merged?

Yes, I'm willing to make the changes required

**Comment by codespearhead** [comment]
> > @mansandersson are you still willing to act on the changes and have this PR merged?
> 
> Yes, I'm willing to make the changes required

Understood. Let me know if you need help! 😄

**Comment by easingthemes** [comment]
Thanks.
Just one more: resolve conflicts in dist. Remove `dist/index.js` and rebuild it locally, then push new version.
Pleas use `fix: some message` in commit message so that release can be automatically triggered.

**Comment by mansandersson** [comment]
> Thanks. Just one more: resolve conflicts in dist. Remove `dist/index.js` and rebuild it locally, then push new version. Pleas use `fix: some message` in commit message so that release can be automatically triggered.

Done. Let me know if anything else needs fixing.

**Comment by joebnb** [comment]
waiting for merging

**Review by easingthemes — APPROVED** [review]

**Comment by easingthemes** [comment]
Thanks

**Pull request merged**